### PR TITLE
[DOCS-6969] What's New update for ACS 7.3

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Waiting on 5.2 or 5.1 confirmation. Also, not sure why the Note box didn't format properly.

### CODE REVIEW

File: content-services/latest/release/index.md [hunk 1]
```diff
@@ -2,81 +2,72 @@
...
+
+### Support for Alfresco Search Enterprise 3.2 with Elasticsearch and OpenSearch
+
+Enables organizations to easily deploy, use and scale Elasticsearch or OpenSearch with Alfresco.  7.3 builds on 7.2 and further extends support to Elasticsearch and OpenSearch.
 
 ## Deprecated software
-* AWS Glacier Connector (replaced with AWS S3 Connector 5.n)
-* Alfresco Content Services Mobile Application
+
+We are ending support for the Share based module for SAML and replacing it with an Identity Service (IDS) based solution, that will greatly improve the customer SSO experience.
```
Comment (vprovaggi) [inline content-services/latest/release/index.md:38]:
```suggestion
We are ending support for the Alfresco SAML SSO module, previously used to configure SAML authentication in Alfresco Office Service (AOS). Coherently with other components in Alfresco, the new solution is now based on an Identity Service (IDS) via OAuth2 that will greatly improve the customer SSO experience. Configuring SAML will still be possible but as part of identity provider configuration.
```

File: content-services/latest/release/index.md [hunk 2]
```diff
@@ -2,81 +2,72 @@
...
 
 Here's a quick summary of what's new in the Alfresco Content Services 7.3 release.
 
 ## Highlights
-* **Support for Direct Access URLs** 
-* **New storage capabilities for Cloud integrations** 
-* **Support for Alfresco Search Enterprise 3.1 with Elasticsearch**
+
+* **New License**
+* **Activiti Console removed**
```
[the remark this answers is not in the document]
Comment (HeinRagas) [reply content-services/latest/release/index.md:14]: Let's go with the following order:
- Java 17 support
- Search Enterprise 2.3 support
- New License
- Activiti console removal.

I've also changed the order in the PDF release notes.

[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply content-services/latest/release/index.md:14]: @HeinRagas I've updated the order, but can you link the updated release notes in the DOCS ticket please? I want to confirm the Search Enterprise support version and heading to use.

**Review by HeinRagas — APPROVED** [review]
Yes, no further comment!

File: content-services/latest/release/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (anxumalo) [reply content-services/latest/release/index.md]: Search Enterprise 3.2 is the correct version 👍🏼

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**